docs/tutorials/basics/60_methods.md

```diff
@@ -107,3 +107,89 @@ say_hello 3
 
 Overloading isn't defined just by type restrictions. The number of arguments as well as named arguments are also
 relevant characteristics.
+
+## Returning a value
+
+Methods may also return a value:
+
+```crystal-play
+def adds_2(n : Int32)
+  n + 2
+end
+
+puts adds_2 40
+```
+
+!!! note
+    In the previous example, the use of the keyword `return` in method `adds_2` would be redundant since methods always return the evaluation of the last line.
+
+The following example illustrates the use of an _explicit_ and an _implicit_ `return`:
+
+```crystal-play
+# This method returns:
+# - the same number if it's even,
+# - the number multiplied by 2 if it's odd.
+def build_even_number(n : Int32)
+  return n if n.even?
+
+  times_2(n)
+end
+
+def times_2(n : Int32) : Int32
+  n * 2
+end
+
+puts build_even_number 7
+puts build_even_number 28
+```
+
+### Return type
+
+As we can see in the previous example, in `times_2` definition we provide type information for the returned value. Here is another example:
+
+```crystal-play
+def hello_message_for(recipient : String) : String
+  "Hello #{recipient}!"
+end
+
+puts hello_message_for "Crystal"
+```
+
+This is really useful for finding errors at compile time:
+
+```crystal
+def hello_message_for(recipient : String) : String
+  "Hello #{recipient}!"
+  42
+end
+
+hello_message_for "Crystal" # => Error: method top-level hello_message_for must return String but it is returning Int32
```

Comment:
Ditto. This example even becomes much more concise by dropping the entire string literal.
```suggestion
def hello_message : String
  42
end

hello_message_for "Crystal" # Error: method top-level hello_message must return String but it is returning Int32
```